### ISSUE TITLE
NEP9 Scan Fail: NEO & GAS

### ISSUE THREAD

**Issue by nickfujita** [O3Labs/O3Android#171]
There appears to be a bug when scanning NEP9 QR codes that contain the asset value NEO or GAS. 

Scan NEP9 QR code:
- neo:<address>?asset=NEO

Expected result
- populate send screen fields with address and NEO

Actual result
- no fields are populated

The scanner still works for scanning NEP9 QR codes with NEP5 tokens, so it seems to be specific to NEO and GAS.

**Comment by saltyskip** [comment]
Yes, I think this is because I was using assed ID for the UTXO assets too, instead of the name. I noticed it, and it has been fixed in 1.11, but I will double check one more time

**Comment by saltyskip** [comment]
I looked at this again and misread it last time and have some questions. Why would address field be populated with neo:?asset=NEO?

Not sure if this is typo. If asset is only meant to be populated I think it doesn't apply to new send flow.

Which goes where -> what -> review.

If only asset is filled in you would end up on the what field, and it would have to go back to fill in address?

**Comment by nickfujita** [comment]
it was a markdown issue. seems to not like < or >

![image](https://user-images.githubusercontent.com/6342442/42756075-94b01084-8935-11e8-8dfd-a367fdda0114.png)

**Comment by saltyskip** [comment]
Gotcha, this is fixed in 1.11
